region/maridia/inner-green.json

```diff
@@ -2561,6 +2561,22 @@
                     "canSpaceJumpWaterBounce"
                   ],
                   "note": "Space jump while partially submerged for more time to mid-air morph."
+                },
+                {
+                  "name": "Spring Ball Room Escape - Suitless, Bootless, Space Jump",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "canSuitlessMaridia",
+                    "canSpringBallJumpMidAir",
+                    "canSunkenTileWideWallClimb",
+                    "canTrickyJump",
+                    "canSpaceJumpWaterBounce"
+                  ],
+                  "note": [
+                    "Position Samus to consecutive wall jump without moving horizontally to perform many wall jumps to get to the water surface.",
```

Comment:
Not sure what this means.  `canSunkenTileWideWallClimb` is done by holding forward into a wall.  Once you're waterBouncing you do have to stop walljumping I suppose.